diff --git a/node_modules/ssml-tsx/dist/index.js b/node_modules/ssml-tsx/dist/index.js
index 9076036..139d708 100644
--- a/node_modules/ssml-tsx/dist/index.js
+++ b/node_modules/ssml-tsx/dist/index.js
@@ -21,3 +21,5 @@ const ssml = (type, props, ...children) => createElement_1.createElement(type, p
 ssml.renderToString = renderToString_1.renderToString;
 ssml.createElement = createElement_1.createElement;
 exports.default = ssml;
+
+module.exports = ssml;
\ No newline at end of file
